src/Pages/Main/Components/MainSectionComponent/Components/MainSectionSlider.js

```diff
@@ -0,0 +1,209 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { flexcenter } from "../../../../../styles/Theme";
+import { FaPlayCircle } from "react-icons/fa";
+import { MdKeyboardArrowLeft, MdKeyboardArrowRight } from "react-icons/md";
+import MainCardCollection from "./MainCardCollection";
+import { BsDot } from "react-icons/bs";
+
+const MainSectionSlider = () => {
+  const [data, setData] = useState([]);
+  const [realData, setRealData] = useState([]);
+  const [x, setX] = useState(0);
+  const FIRST_API = "http://10.168.1.52:8000/music/album/info";
+  const [current, setCurrent] = useState(0);
+  const length = data.length;
+
+  React.useEffect(() => {
+    fetch("/data/aramData.json")
+      .then(res => res.json())
+      .then(res => setData(res.songs1));
+  }, []);
+
+  const prevSlide = () => {
+    setX(x + 105.7);
+
+    setCurrent(current === 0 ? length - 1 : current - 1);
+  };
+  const nextSlide = () => {
+    setX(x - 105.7);
+    setCurrent(current === length - 1 ? 0 : current + 1);
+  };
+  const dotFunction = num => {
+    if (num === 0) {
+      setX(0);
+      setCurrent(0);
+      console.log(current);
+    }
+    if (num === 1) {
+      setX(-105.5);
+      setCurrent(1);
+      console.log(current);
+    }
+    if (num === 2) {
+      setX(-211);
+      setCurrent(2);
+      console.log(current);
+    }
+    if (num === 3) {
+      setX(-316.5);
+      setCurrent(3);
+      console.log(current);
+    }
+    if (num === 4) {
+      setX(-422);
+      setCurrent(4);
+      console.log(current);
+    }
+  };
```

Comment:
x 변동에 인자에 따라서 규칙이 있지 않을까요? 아래와 같이 할 수 있을 것 같은데 확인 부탁드립니다
```suggestion
  const dotFunction = num => {
    const newX = num을 사용한 값
    
    set(newX)
    setCurrent(num)
  };
```